database/migrations/2020_01_01_140000_create_keywords_table.php

```diff
@@ -14,6 +14,7 @@ public function up()
             $table->id();
             $table->string('phrase')->index()->unique();
             $table->string('type')->index(); // Example: 'Branded', 'Generic', 'Local'
+            $table->boolean('is_active')->index();
```

Comment:
I would prefer to not have this in the database. Can we just create a scope for the active keywords based on the datetime fields for `tracking_requested_at` and `tracking_stopped_at`?

Reply:
Yes I will put that back.

Reply:
Thanks!

Let's discuss with the team and we may want to add this back if there are performance gains.

CC: @pdbreen @jasonmccreary @joshtorres @phuclh

Reply:
Sorry @drewroberts I had asked Arwa to add this since we are going to be using that active scope for everything in regards to keywords. Yes I was thinking it would be better for performance but that just depends on how big the keyword database will get. If it's not going to get that big then it probably wouldn't make much of a difference. I am curious to know the others' opinions on it though.